<!DOCTYPE html>
<html lang="en-US" dir="ltr">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>Bloom Lab</title>
    <meta name="description" content="We study the evolution of viruses.">
    <meta name="generator" content="VitePress v1.1.0">
    <link rel="preload stylesheet" href="/assets/style.BJoSVSgr.css" as="style">
    
    <script type="module" src="/assets/app.CcfNRZmG.js"></script>
    <link rel="modulepreload" href="/assets/chunks/framework.D4bY2S_W.js">
    <link rel="modulepreload" href="/assets/chunks/theme.DwK3Lct_.js">
    <link rel="modulepreload" href="/assets/index.md.CplNLCOO.lean.js">
    <link rel="icon" href="/images/icon.png">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
    <script id="check-mac-os">document.documentElement.classList.toggle("mac",/Mac|iPhone|iPod|iPad/i.test(navigator.platform));</script>
  </head>
  <body>
    <div id="app"><div id="app" class="flex flex-col min-h-screen"><nav class="p-4 mb-4"><div class="flex justify-between items-center w-full px-4 select-none"><a href="/" class="flex items-center no-underline"><img src="/images/icon.png" alt="Logo" class="h-8 w-8 mr-2"><span class="title text-lg font-bold text-custom-orange collapse sm:visible">Bloom Lab</span></a><button class="flex md:hidden text-gray-600"><span class="i-menu h-6 w-6"></span></button><div class="flex bg-custom-soft backdrop-blur-md md:backdrop-blur-none flex-col md:flex-row bg-opacity-75 md:bg-transparent md:flex md:items-center md:px-0 px-3 md:pb-0 pb-10 md:static absolute md:w-auto w-full md:h-auto h-full md:pt-0 pt-10 top-14 z-50 md:flex hidden left-[-100%]"><!--[--><a href="/posts/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Blog</a><a href="/papers/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Papers</a><a href="/people/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Team</a><a href="/projects/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Software</a><!--]--></div></div></nav><main class="flex-1 max-w-3xl mx-auto px-4 sm:px-6 lg:max-w-5xl xl:px-0"><div class="hide-scrollbar snap-y snap-mandatory overflow-y-scroll overscroll-contain h-screen"><div class="hide-scrollbar snap-always snap-start mx-auto max-w-4xl px-4 text-left flex flex-col justify-between h-screen overflow-scroll"><div class="flex-grow"><h1 class="text-4xl sm:text-5xl md:text-7xl lg:text-8xl text-gray-800 select-none"> Welcome to the <span class="block font-bold">Bloom Lab</span></h1><p class="text-xl md:text-2xl lg:text-3xl text-gray-600 mt-2 lg:mt-4 select-none"> where we study the evolution of <span class="italic" style="color:#000000;background-color:rgba(0, 0, 0, 0.1);">Viruses</span></p><p class="text-sm md:text-base lg:text-lg text-gray-500 mt-4 md:mt-6 lg:mt-12"> In the <strong>Bloom lab</strong>, we study the evolution of viruses and proteins. We use experimental and computational techniques to understand the molecular constraints on viral proteins, and how these constraints shape the capacity of viruses to evolve to escape antibodies, erode pre-existing immunity, and adapt to new hosts. </p><p class="text-sm md:text-base lg:text-lg text-gray-500 mt-2 md:mt-4 lg:mt-6"> Our lab is part of the <a href="https://www.fredhutch.org/en/research/divisions/basic-sciences-division.html">Basic Sciences Divison</a> and the <a href="https://www.fredhutch.org/en/research/divisions/public-health-sciences-division/research/computational-biology.html">Computational Biology Program</a> at the Fred Hutch. We are also affiliated with the <a href="https://www.gs.washington.edu/">Genome Sciences</a> and <a href="https://microbiology.washington.edu/uw-microbiology-home">Microbiology</a> Departments at the University of Washington, and graduate students often join our lab through the <a href="https://mcb-seattle.edu">MCB program</a>. Jesse Bloom is an Investigator of the <a href="https://www.hhmi.org/research">Howard Hughes Medical Institute</a>. </p><div class="scroll-button pt-2"><div><p class="text-sm md:text-base lg:text-lg text-gray-500 mt-10 md:mt-6 lg:mt-12 text-center"> Scroll down to learn more </p><div class="flex justify-center"><i class="bi bi-chevron-down text-gray-500 mt-1 animate-bounce text-lg md:text-xl lg:text-2xl"></i></div></div></div></div></div><div class="research-aims pb-40"><!--[--><div class="mx-auto max-w-4xl px-4 text-left flex flex-col justify-between"><h2 class="sky snap-always snap-start text-2xl md:text-3xl lg:text-4xl text-gray-600 mt-4 md:mt-6 lg:mt-12 mb-2 select-none underline font-semibold">Deep Mutational Scanning</h2><div class="aim-content"><p>Our lab uses deep mutational scanning to experimentally measure how tens-of-thousands of mutations to viral proteins affect key properties including function, immune escape, and receptor binding.</p>
<p><img src="/assets/research/dms.png" alt="&quot;Pseudovirus neutralization system&quot;"></p>
<p>We primarily perform these experiments using a <a href="https://www.sciencedirect.com/science/article/pii/S0092867423001034" target="_blank" rel="noreferrer">pseudovirus system</a> that allows us to safely characterize mutants of entry proteins from a wide range of viruses, including <a href="https://www.nature.com/articles/s41586-024-07636-1" target="_blank" rel="noreferrer">SARS-CoV-2 spike</a>, <a href="https://doi.org/10.1101/2024.05.23.595634" target="_blank" rel="noreferrer">influenza hemagglutinin</a>, <a href="https://www.sciencedirect.com/science/article/pii/S1074761324003194" target="_blank" rel="noreferrer">Lassa virus GPC</a>, <a href="https://www.sciencedirect.com/science/article/pii/S1931312823002184" target="_blank" rel="noreferrer">HIV envelope</a>, and <a href="https://doi.org/10.1101/2024.04.17.589977" target="_blank" rel="noreferrer">Nipah virus RBP</a>.</p>
<p>Deep mutational scanning can inform efforts to <a href="https://www.nature.com/articles/s41586-024-07636-1" target="_blank" rel="noreferrer">forecast the evolution of human seasonal viruses</a> and <a href="https://doi.org/10.1101/2024.05.23.595634" target="_blank" rel="noreferrer">surveil the evolution of potential pandemic viruses</a>. To facilitate the use of deep mutational scanning for these important goals, we develop <a href="https://dms-viz.github.io/" target="_blank" rel="noreferrer">interactive visualization tools</a> and <a href="https://github.com/dms-vep/dms-vep-pipeline-3" target="_blank" rel="noreferrer">data analysis pipelines</a>. See <a href="https://dms-vep.org/Flu_H5_American-Wigeon_South-Carolina_2021-H5N1_DMS/" target="_blank" rel="noreferrer">here</a> for an example of how we analyze and visualize large datasets to inform the study of viral evolution.</p>
</div></div><div class="mx-auto max-w-4xl px-4 text-left flex flex-col justify-between"><h2 class="indigo snap-always snap-start text-2xl md:text-3xl lg:text-4xl text-gray-600 mt-4 md:mt-6 lg:mt-12 mb-2 select-none underline font-semibold">Interplay of Immunity and Viral Evolution</h2><div class="aim-content"><p>We study immunity and viral evolution at both the population and single-cell levels.</p>
<p><img src="/assets/research/seqneut.jpg" alt="&quot;Sequencing-based neutralization assay&quot;"></p>
<p>At the population level, differences in exposure history and immune imprinting lead human individuals to make antibody responses that target different regions of rapidly evolving viruses like influenza and SARS-CoV-2. This population heterogeneity has profound implications for viral evolution and disease susceptibility, as it causes viral mutations to impact the immunity of different individuals differently. We are characterizing this population heterogeneity using both <a href="https://www.sciencedirect.com/science/article/pii/S1931312824002336" target="_blank" rel="noreferrer">deep mutational scanning</a> and a <a href="https://doi.org/10.1101/2024.03.08.584176" target="_blank" rel="noreferrer">sequencing based-neutralization assay</a> we developed that increases the throughput of traditional neutralization assays by several orders of magnitude (see schematic at left).</p>
<p>At the single-cell level, we developed approaches to <a href="https://journals.asm.org/doi/10.1128/jvi.00500-19" target="_blank" rel="noreferrer">sequence viruses in single cells</a> and <a href="https://elifesciences.org/articles/86852" target="_blank" rel="noreferrer">quantify how many progeny each infected cell produces</a>. We use these approaches to understand how viral variation impacts the outcome of infection in individual cells.</p>
</div></div><div class="mx-auto max-w-4xl px-4 text-left flex flex-col justify-between"><h2 class="pink snap-always snap-start text-2xl md:text-3xl lg:text-4xl text-gray-600 mt-4 md:mt-6 lg:mt-12 mb-2 select-none underline font-semibold">Big Datasets and Viral Evolution</h2><div class="aim-content"><p>We also develop new ways to leverage large datasets to better understand viral evolution.</p>
<p>We have come up with a way to leverage the millions of publicly available SARS-CoV-2 sequences to estimate the effect of individual mutations on viral fitness (see <a href="https://academic.oup.com/ve/article/9/2/vead055/7265011" target="_blank" rel="noreferrer">this paper</a> and <a href="https://slides.com/jbloom/sars2-mut-fitness/" target="_blank" rel="noreferrer">these slides</a>). We've also created a platform to <a href="https://jbloomlab.github.io/SARS2-mut-fitness/" target="_blank" rel="noreferrer">visualize the mutational effects</a> to aid in interpretation of viral evolution.</p>
<p>We have also integrated thousands of deep mutational scanning measurements into an <a href="https://jbloomlab.github.io/SARS2-RBD-escape-calc/" target="_blank" rel="noreferrer">antibody-escape calculator</a> that was <a href="https://slides.com/jbloom/sars2-rbd-escape-calc#/12" target="_blank" rel="noreferrer">widely used</a> during the SARS-CoV-2 pandemic to understand the antigenic effects of viral mutations.</p>
<p>We also have projects that involve analyzing the evolution of viruses <a href="https://elifesciences.org/articles/26875" target="_blank" rel="noreferrer">within individual infected humans</a>, and developing models to <a href="https://academic.oup.com/ve/article/8/2/veac110/6889254" target="_blank" rel="noreferrer">understand epistasis among viral mutations</a>.</p>
</div></div><!--]--></div></div></main><footer class="w-full p-4 bg-white text-center text-gray-400 text-sm"><p> Copyright © Jesse Bloom 2024-Present </p></footer></div></div>
    <script>window.__VP_HASH_MAP__=JSON.parse("{\"papers_2015_doud.md\":\"48Gi8XhC\",\"papers_2015_hooper.md\":\"CI0Ar8ga\",\"papers_2015_machkovech.md\":\"Bf5MUNip\",\"people_lizzie-plender.md\":\"CrKX8dse\",\"people_lucas-kampman.md\":\"BCZ-Tnni\",\"papers_2021_greaney_welsh.md\":\"DNVgkeUO\",\"readme.md\":\"D8eBG4ob\",\"index.md\":\"CplNLCOO\",\"papers_2013_gong.md\":\"PyD3I5qV\",\"papers_2013_hooper.md\":\"Cd1R1SDU\",\"papers_2014_bloom_a.md\":\"TW6XA02c\",\"papers_2014_bloom_b.md\":\"Cm1oDw8Z\",\"papers_2015_bloom.md\":\"D3hHe5R3\",\"papers_2016_doud.md\":\"DkRFPPp6\",\"papers_2016_haddox.md\":\"nBWTXwbj\",\"papers_2016_xue.md\":\"8q0P6h_J\",\"papers_2017_ashenberg.md\":\"Df9IJNfu\",\"papers_2017_bloom.md\":\"CGsqWTko\",\"papers_2017_dingens.md\":\"Cbmr6FGX\",\"papers_2017_doud.md\":\"CIblnIx8\",\"papers_2017_hilton.md\":\"CboQGh0y\",\"papers_2017_xue.md\":\"DskBsQGR\",\"papers_2018_dingens.md\":\"D5n97hEY\",\"papers_2018_doud.md\":\"mfpzn1GV\",\"papers_2018_haddox.md\":\"-EzsKUxl\",\"papers_2018_hilton.md\":\"C49rLQnc\",\"papers_2018_lee.md\":\"BRV6Fb_g\",\"papers_2018_russell.md\":\"wgx_bish\",\"papers_2018_xue_a.md\":\"CEdqDxk1\",\"papers_2018_xue_b.md\":\"KLBeer-G\",\"papers_2019_crawford.md\":\"s7R-zLVQ\",\"papers_2019_dingens_a.md\":\"IrHjZDnk\",\"papers_2019_dingens_b.md\":\"C51xx5_b\",\"papers_2019_lee.md\":\"BHzJxPUH\",\"papers_2019_machkovech.md\":\"lJ_oB7Fz\",\"papers_2019_russell.md\":\"CDGyoaHs\",\"papers_2019_soh.md\":\"Bf5D9w2H\",\"papers_2019_xue_a.md\":\"BmTcPROU\",\"papers_2020_crawford.md\":\"COKdMTcD\",\"papers_2020_dingens.md\":\"bB3YILP3\",\"papers_2020_einav_b.md\":\"Cf2gD3fe\",\"papers_index.md\":\"A1_cAYTX\",\"papers_2020_gentles.md\":\"CU4t53Mc\",\"people_andrea-loes.md\":\"Dd1vld5d\",\"papers_2020_hilton.md\":\"Dq9V-1yM\",\"people_andrew-butler.md\":\"DKn4H4UK\",\"people_arjun-aditham.md\":\"BhsY8v3S\",\"papers_2020_loes.md\":\"CmmoyyvP\",\"people_bernadeta-dadonaite.md\":\"BMZ07Bgq\",\"papers_2020_roop.md\":\"BpQAw3YD\",\"people_brendan-larsen.md\":\"2bJRg1_l\",\"people_caelan-radford.md\":\"xo7jW8eT\",\"papers_2020_starr_a.md\":\"Cjc4mid6\",\"people_caleb-carr.md\":\"DQ-itn5i\",\"papers_2020_xue.md\":\"tetMVX9z\",\"papers_2021_bloom_a.md\":\"B20P_Vtx\",\"people_cassandra-simonich.md\":\"BETrCWbs\",\"papers_2021_crawford.md\":\"LqRx9oAP\",\"people_index.md\":\"Bgrxj6K-\",\"papers_2021_dingens.md\":\"BpRmlWAS\",\"people_jenny-ahn.md\":\"DzArYfTQ\",\"papers_2021_eguia.md\":\"BxoLaVWQ\",\"people_jesse-bloom.md\":\"BBI88SUf\",\"people_sheri-harari.md\":\"BFT1hw-7\",\"papers_2021_greaney_a.md\":\"tqZcoDO4\",\"people_teagan-mcmahon.md\":\"DvwHjcmC\",\"people_caroline-kikawa.md\":\"DwYRhm3r\",\"papers_2021_greaney_b.md\":\"C70H2abN\",\"people_tim-yu.md\":\"A1grVJtf\",\"papers_2021_greaney_c.md\":\"DOMukqxr\",\"people_wenlin-ren.md\":\"BPzf39Md\",\"papers_2021_greaney_d.md\":\"B1gyjjsj\",\"papers_2021_starr_b.md\":\"BDv6UmMj\",\"papers_2021_starr_greaney.md\":\"IB3VJQQN\",\"papers_2014_gong.md\":\"D6j_RWFK\",\"papers_2021_starr_greaney_a.md\":\"CMhFAqkF\",\"papers_2022_farrell.md\":\"JaYpeG8h\",\"papers_2022_gentles.md\":\"BP2KThjj\",\"papers_2022_greaney_a.md\":\"CiHEmU10\",\"papers_2022_greaney_b.md\":\"DWzt8_d0\",\"papers_2022_greaney_c.md\":\"BtKGaOA7\",\"papers_2021_soh.md\":\"BI0dWn0D\",\"papers_2022_hannon.md\":\"CmZdo8Lg\",\"papers_2022_starr_a.md\":\"S2Zs3F-f\",\"papers_2022_starr_greaney_a.md\":\"D5j0_yFr\",\"papers_2024_carr.md\":\"OihxBD-v\",\"papers_2022_starr_greaney_b.md\":\"DNdvVrrg\",\"papers_2022_yu.md\":\"kTo3Hr8f\",\"papers_2014_thyagarajan.md\":\"Dx5eh_Ub\",\"people_sara-sunshine.md\":\"BUrrV13b\",\"papers_2023_bacsik.md\":\"CSLMWJ8m\",\"papers_2023_bloom_a.md\":\"BCf1Dh2c\",\"research_big-data-and-viruses.md\":\"TC2M2KwD\",\"research_deep-mutational-scanning.md\":\"ChHBytu2\",\"papers_2023_bloom_b.md\":\"DAvFQSAm\",\"research_immunity-and-evolution.md\":\"CARD5jyA\",\"papers_2023_bloom_c.md\":\"YecZKxe_\",\"papers_2023_dadonaite_crawford_radford.md\":\"CjEimFsm\",\"papers_2013_ashenberg.md\":\"DQMwPcgT\",\"papers_2023_kikawa.md\":\"sK6GyFCo\",\"projects_alignparse.md\":\"CBU5zvZR\",\"papers_2023_radford.md\":\"C_kQNWwR\",\"projects_dms-vep-pipeline-3.md\":\"Dlh9ZFyk\",\"papers_2024_bloom.md\":\"BKiXUzLp\",\"projects_dms-viz.md\":\"BuCdXmdr\",\"projects_index.md\":\"BYXuju14\",\"papers_2024_dadonaite_a.md\":\"BJ4gMuJx\",\"projects_neutcurve.md\":\"Dco99HIa\",\"papers_2024_dadonaite_b.md\":\"CzgcmbCI\",\"projects_polyclonal.md\":\"DVhxAPLQ\",\"papers_2024_hannon.md\":\"Dd9QC8Cw\",\"projects_seqneut-pipeline.md\":\"Ch4cifq3\",\"papers_2024_larsen.md\":\"CdCk92Pt\",\"posts_index.md\":\"CKU23Rsq\",\"people_xiaohui-ju.md\":\"wVgHbx0Q\",\"papers_2024_loes.md\":\"Ctn4IIjR\",\"papers_2024_welsh.md\":\"DE5cBDQg\",\"people_will-hannon.md\":\"BnukX3BJ\",\"posts_h5-dms.md\":\"DJbO7pAW\"}");window.__VP_SITE_DATA__=JSON.parse("{\"lang\":\"en-US\",\"dir\":\"ltr\",\"title\":\"Bloom Lab\",\"description\":\"We study the evolution of viruses.\",\"base\":\"/\",\"head\":[],\"router\":{\"prefetchLinks\":true},\"appearance\":false,\"themeConfig\":{},\"locales\":{},\"scrollOffset\":134,\"cleanUrls\":false}");</script>
    
  </body>
</html>