import{_ as t,c as i,o as a,b as e,d as n}from"./chunks/framework.B63u0Dcf.js";const v=JSON.parse('{"title":"Reconciling disparate estimates of viral genetic diversity during human influenza infections","description":"","frontmatter":{"layout":"paper","title":"Reconciling disparate estimates of viral genetic diversity during human influenza infections","date":"2019-09-01","authors":["Katherine S Xue","Jesse D Bloom"],"journal":"Nature genetics","doi":"10.1038/s41588-019-0349-3","link":"https://www.nature.com/articles/s41588-019-0349-3","image":"/assets/papers/2019_xue_a.jpg","keywords":["Influenza","Within-host evolution"],"head":[["link",{"rel":"canonical","href":"https://jbloomlab.org/papers/2019_xue_a.html"}]]},"headers":[],"relativePath":"papers/2019_xue_a.md","filePath":"papers/2019_xue_a.md"}'),s={name:"papers/2019_xue_a.md"},r=e("h2",{id:"abstract",tabindex:"-1"},[n("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"​")],-1),o=e("p",null,"To the Editor — A key question in the study of influenza virus evolution is how rapidly viral genetic variation arises within infected humans1,2. Recently, several studies have measured influenza’s within-host genetic diversity in large cohorts of infected humans through high-throughput deep sequencing3,4,5,6 (Supplementary Table 1). These studies disagree in their estimates of influenza’s within-host genetic diversity. In a Nature Genetics letter titled ‘Quantifying influenza virus diversity and transmission in humans’, analyzing a household cohort in Hong Kong, Poon et al.4 have estimated that within-host genetic diversity is high, and 200–250 viral genomes are transmitted between individuals. However, several recent studies conducted in Wisconsin3, Michigan6, and Washington7 that used similar methodologies have estimated lower levels of viral genetic diversity. In particular, the Michigan study has estimated a narrow transmission bottleneck of just one or two viral genomes6. We sought to examine whether technical differences in the underlying deep-sequencing datasets or the methods used to analyze them might explain the disparate estimates of within-host viral genetic diversity. We identified an anomaly in the Hong Kong data that provides a technical explanation for these discrepancies: read pairs from this study are often split between different biological samples, thus indicating that some reads are incorrectly assigned.",-1),d=[r,o];function h(l,c,u,m,p,g){return a(),i("div",null,d)}const _=t(s,[["render",h]]);export{v as __pageData,_ as default};
