import{_ as t,c as o,o as i,b as e,d as n}from"./chunks/framework.B63u0Dcf.js";const v=JSON.parse('{"title":"Site-specific amino acid preferences are mostly conserved in two closely related protein homologs","description":"","frontmatter":{"layout":"paper","title":"Site-specific amino acid preferences are mostly conserved in two closely related protein homologs","date":"2015-11-01","authors":["Michael B Doud","Orr Ashenberg","Jesse D Bloom"],"journal":"Molecular biology and evolution","doi":"10.1093/molbev/msv167","link":"https://academic.oup.com/mbe/article/32/11/2944/982113","image":"/assets/papers/2015_doud.jpg","keywords":["Influenza","Epistasis","Deep mutational scanning"]},"headers":[],"relativePath":"papers/2015_doud.md","filePath":"papers/2015_doud.md"}'),a={name:"papers/2015_doud.md"},s=e("h2",{id:"abstract",tabindex:"-1"},[n("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"​")],-1),r=e("p",null,"Evolution drives changes in a protein’s sequence over time. The extent to which these changes in sequence lead to shifts in the underlying preference for each amino acid at each site is an important question with implications for comparative sequence-analysis methods, such as molecular phylogenetics. To quantify the extent that site-specific amino acid preferences shift during evolution, we performed deep mutational scanning on two homologs of human influenza nucleoprotein with 94% amino acid identity. We found that only a modest fraction of sites exhibited shifts in amino acid preferences that exceeded the noise in our experiments. Furthermore, even among sites that did exhibit detectable shifts, the magnitude tended to be small relative to differences between nonhomologous proteins. Given the limited change in amino acid preferences between these close homologs, we tested whether our measurements could inform site-specific substitution models that describe the evolution of nucleoproteins from more diverse influenza viruses. We found that site-specific evolutionary models informed by our experiments greatly outperformed nonsite-specific alternatives in fitting phylogenies of nucleoproteins from human, swine, equine, and avian influenza. Combining the experimental data from both homologs improved phylogenetic fit, partly because measurements in multiple genetic contexts better captured the evolutionary average of the amino acid preferences for sites with shifting preferences. Our results show that site-specific amino acid preferences are sufficiently conserved that measuring mutational effects in one protein provides information that can improve quantitative evolutionary modeling of nearby homologs.",-1),c=[s,r];function l(d,m,h,p,u,f){return i(),o("div",null,c)}const b=t(a,[["render",l]]);export{v as __pageData,b as default};
