import{_ as t,c as a,o as s,b as e,d as o}from"./chunks/framework.BF-rJS87.js";const b=JSON.parse('{"title":"Software for the analysis and visualization of deep mutational scanning data","description":"","frontmatter":{"layout":"paper","title":"Software for the analysis and visualization of deep mutational scanning data","date":"2015-12-01","authors":["Jesse D Bloom"],"journal":"BMC bioinformatics","doi":"10.1186/s12859-015-0590-4","link":"https://link.springer.com/article/10.1186/s12859-015-0590-4","image":"/assets/papers/2015_bloom.jpg","keywords":["Software tools"]},"headers":[],"relativePath":"papers/2015_bloom.md","filePath":"papers/2015_bloom.md"}'),n={name:"papers/2015_bloom.md"},i=e("h2",{id:"abstract",tabindex:"-1"},[o("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"â€‹")],-1),r=e("p",null,"Deep mutational scanning is a technique to estimate the impacts of mutations on a gene by using deep sequencing to count mutations in a library of variants before and after imposing a functional selection. The impacts of mutations must be inferred from changes in their counts after selection.",-1),c=e("p",null,"I describe a software package, dms_tools, to infer the impacts of mutations from deep mutational scanning data using a likelihood-based treatment of the mutation counts. I show that dms_tools yields more accurate inferences on simulated data than simply calculating ratios of counts pre- and post-selection. Using dms_tools, one can infer the preference of each site for each amino acid given a single selection pressure, or assess the extent to which these preferences change under different selection pressures.",-1),l=[i,r,c];function d(m,p,f,u,h,_){return s(),a("div",null,l)}const k=t(n,[["render",d]]);export{b as __pageData,k as default};
