import{_ as t,c as a,o as s,b as e,d as i}from"./chunks/framework.BF-rJS87.js";const _=JSON.parse('{"title":"Extreme heterogeneity of influenza virus infection in single cells","description":"","frontmatter":{"layout":"paper","title":"Extreme heterogeneity of influenza virus infection in single cells","date":"2018-02-16","authors":["Alistair B Russell","Cole Trapnell","Jesse D Bloom"],"journal":"eLife","doi":"10.7554/eLife.32303","link":"https://elifesciences.org/articles/32303","image":"/assets/papers/2018_russell.jpg","keywords":["Influenza","Single-cell sequencing"]},"headers":[],"relativePath":"papers/2018_russell.md","filePath":"papers/2018_russell.md"}'),l={name:"papers/2018_russell.md"},n=e("h2",{id:"abstract",tabindex:"-1"},[i("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"​")],-1),r=e("p",null,"Viral infection can dramatically alter a cell’s transcriptome. However, these changes have mostly been studied by bulk measurements on many cells. Here we use single-cell mRNA sequencing to examine the transcriptional consequences of influenza virus infection. We find extremely wide cell-to-cell variation in the productivity of viral transcription – viral transcripts comprise less than a percent of total mRNA in many infected cells, but a few cells derive over half their mRNA from virus. Some infected cells fail to express at least one viral gene, but this gene absence only partially explains variation in viral transcriptional load. Despite variation in viral load, the relative abundances of viral mRNAs are fairly consistent across infected cells. Activation of innate immune pathways is rare, but some cellular genes co-vary in abundance with the amount of viral mRNA. Overall, our results highlight the complexity of viral infection at the level of single cells.",-1),o=[n,r];function c(u,f,p,d,m,h){return s(),a("div",null,o)}const g=t(l,[["render",c]]);export{_ as __pageData,g as default};
