import{_ as t,c as a,o,b as e,d as n}from"./chunks/framework.O46bkOJt.js";const b=JSON.parse('{"title":"An experimentally informed evolutionary model improves phylogenetic fit to divergent lactamase homologs","description":"","frontmatter":{"layout":"paper","title":"An experimentally informed evolutionary model improves phylogenetic fit to divergent lactamase homologs","date":"2014-10-01","authors":["Jesse D Bloom"],"journal":"Molecular biology and evolution","doi":"10.1093/molbev/msu220","link":"https://academic.oup.com/mbe/article/31/10/2753/1015257","image":"/assets/papers/2014_bloom_a.jpg","keywords":["Phylogenetics"],"head":[["link",{"rel":"canonical","href":"https://jbloomlab.org/papers/2014_bloom_a.html"}]]},"headers":[],"relativePath":"papers/2014_bloom_a.md","filePath":"papers/2014_bloom_a.md"}'),s={name:"papers/2014_bloom_a.md"},i=e("h2",{id:"abstract",tabindex:"-1"},[n("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"​")],-1),r=e("p",null,"Phylogenetic analyses of molecular data require a quantitative model for how sequences evolve. Traditionally, the details of the site-specific selection that governs sequence evolution are not known a priori, making it challenging to create evolutionary models that adequately capture the heterogeneity of selection at different sites. However, recent advances in high-throughput experiments have made it possible to quantify the effects of all single mutations on gene function. I have previously shown that such high-throughput experiments can be combined with knowledge of underlying mutation rates to create a parameter-free evolutionary model that describes the phylogeny of influenza nucleoprotein far better than commonly used existing models. Here, I extend this work by showing that published experimental data on TEM-1 beta-lactamase (Firnberg E, Labonte JW, Gray JJ, Ostermeier M. 2014. A comprehensive, high-resolution map of a gene’s fitness landscape. Mol Biol Evol. 31:1581–1592) can be combined with a few mutation rate parameters to create an evolutionary model that describes beta-lactamase phylogenies much better than most common existing models. This experimentally informed evolutionary model is superior even for homologs that are substantially diverged (about 35% divergence at the protein level) from the TEM-1 parent that was the subject of the experimental study. These results suggest that experimental measurements can inform phylogenetic evolutionary models that are applicable to homologs that span a substantial range of sequence divergence.",-1),l=[i,r];function m(c,h,d,p,u,g){return o(),a("div",null,l)}const y=t(s,[["render",m]]);export{b as __pageData,y as default};
