import{_ as t,c as a,o,b as e,d as n}from"./chunks/framework.BOAZPCR7.js";const b=JSON.parse('{"title":"An experimentally determined evolutionary model dramatically improves phylogenetic fit","description":"","frontmatter":{"layout":"paper","title":"An experimentally determined evolutionary model dramatically improves phylogenetic fit","date":"2014-08-01","authors":["Jesse D Bloom"],"journal":"Molecular Biology and Evolution","doi":"10.1093/molbev/msu173","link":"https://academic.oup.com/mbe/article/31/8/1956/2925795","image":"/assets/papers/2014_bloom_b.jpg","keywords":["Phylogenetics"],"head":[["link",{"rel":"canonical","href":"https://jbloomlab.org/papers/2014_bloom_b.html"}]]},"headers":[],"relativePath":"papers/2014_bloom_b.md","filePath":"papers/2014_bloom_b.md"}'),r={name:"papers/2014_bloom_b.md"},s=e("h2",{id:"abstract",tabindex:"-1"},[n("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"â€‹")],-1),i=e("p",null,"All modern approaches to molecular phylogenetics require a quantitative model for how genes evolve. Unfortunately, existing evolutionary models do not realistically represent the site-heterogeneous selection that governs actual sequence change. Attempts to remedy this problem have involved augmenting these models with a burgeoning number of free parameters. Here, I demonstrate an alternative: Experimental determination of a parameter-free evolutionary model via mutagenesis, functional selection, and deep sequencing. Using this strategy, I create an evolutionary model for influenza nucleoprotein that describes the gene phylogeny far better than existing models with dozens or even hundreds of free parameters. Emerging high-throughput experimental strategies such as the one employed here provide fundamentally new information that has the potential to transform the sensitivity of phylogenetic and genetic analyses.",-1),l=[s,i];function m(c,d,p,h,u,g){return o(),a("div",null,l)}const f=t(r,[["render",m]]);export{b as __pageData,f as default};
