import{_ as t,c as a,o as s,b as e,d as i}from"./chunks/framework.B63u0Dcf.js";const b=JSON.parse('{"title":"Mutational effects on stability are largely conserved during protein evolution","description":"","frontmatter":{"layout":"paper","title":"Mutational effects on stability are largely conserved during protein evolution","date":"2013-12-24","authors":["Orr Ashenberg","Lizhi Ian Gong","Jesse D Bloom"],"journal":"PNAS","doi":"10.1073/pnas.1314781111","link":"https://www.pnas.org/doi/abs/10.1073/pnas.1314781111","image":"/assets/papers/2013_ashenberg.jpg","keywords":["Epistasis","Influenza"],"head":[["link",{"rel":"canonical","href":"https://jbloomlab.org/papers/2013_ashenberg.html"}]]},"headers":[],"relativePath":"papers/2013_ashenberg.md","filePath":"papers/2013_ashenberg.md"}'),n={name:"papers/2013_ashenberg.md"},o=e("h2",{id:"abstract",tabindex:"-1"},[i("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"â€‹")],-1),r=e("p",null,"Protein stability and folding are the result of cooperative interactions among many residues, yet phylogenetic approaches assume that sites are independent. This discrepancy has engendered concerns about large evolutionary shifts in mutational effects that might confound phylogenetic approaches. Here we experimentally investigate this issue by introducing the same mutations into a set of diverged homologs of the influenza nucleoprotein and measuring the effects on stability. We find that mutational effects on stability are largely conserved across the homologs. We reach qualitatively similar conclusions when we simulate protein evolution with molecular-mechanics force fields. Our results do not mean that proteins evolve without epistasis, which can still arise even when mutational stability effects are conserved. However, our findings indicate that large evolutionary shifts in mutational effects on stability are rare, at least among homologs with similar structures and functions. We suggest that properly describing the clearly observable and highly conserved amino acid preferences at individual sites is likely to be far more important for phylogenetic analyses than accounting for rare shifts in amino acid propensities due to site covariation.",-1),l=[o,r];function c(h,d,p,u,f,g){return s(),a("div",null,l)}const y=t(n,[["render",c]]);export{b as __pageData,y as default};
