import{_ as a,c as s,o as t,b as e,d as r}from"./chunks/framework.D4bY2S_W.js";const _=JSON.parse('{"title":"alignparse: A Python package for parsing complex features from high-throughput long-read sequencing","description":"","frontmatter":{"layout":"paper","title":"alignparse: A Python package for parsing complex features from high-throughput long-read sequencing","date":"2019-01-23","authors":["Katharine HD Crawford","Jesse D Bloom"],"journal":"JOSS","doi":"10.21105/joss.01915","link":"https://doi.org/10.21105%2Fjoss.01915","image":"/assets/papers/2019_crawford.png","keywords":["Software tools"]},"headers":[],"relativePath":"papers/2019_crawford.md","filePath":"papers/2019_crawford.md"}'),n={name:"papers/2019_crawford.md"},o=e("h2",{id:"abstract",tabindex:"-1"},[r("Abstract "),e("a",{class:"header-anchor",href:"#abstract","aria-label":'Permalink to "Abstract"'},"â€‹")],-1),i=e("p",null,"Advances in sequencing technology have made it possible to generate large numbers of long, high-accuracy sequencing reads. For instance, the new PacBio Sequel platform can generate hundreds of thousands of high-quality circular consensus sequences in a single run (Hebert et al., 2018; Rhoads & Au, 2015). Good programs exist for aligning these reads for genome assembly (Chaisson & Tesler, 2012; Li, 2018). However, these long reads can also be used for other purposes, such as sequencing PCR amplicons that contain various features of interest. For instance, PacBio circular consensus sequences have been used to identify the mutations in influenza viruses in single cells (Russell et al, 2019), or to link barcodes to gene mutants in deep mutational scanning (Matreyek et al., 2018). For such applications, the alignment of the sequences to the targets may be fairly trivial, but it is not trivial to then parse specific features of interest (such as mutations, unique molecular identifiers, cell barcodes, and flanking sequences) from these alignments.",-1),c=[o,i];function l(u,d,h,p,f,g){return t(),s("div",null,c)}const b=a(n,[["render",l]]);export{_ as __pageData,b as default};
