<!DOCTYPE html>
<html lang="en-US" dir="ltr">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>Reconciling disparate estimates of viral genetic diversity during human influenza infections | Bloom Lab</title>
    <meta name="description" content="We study the evolution of viruses.">
    <meta name="generator" content="VitePress v1.1.0">
    <link rel="preload stylesheet" href="/assets/style.DOJY71uG.css" as="style">
    
    <script type="module" src="/assets/app.BRWkTSJp.js"></script>
    <link rel="modulepreload" href="/assets/chunks/framework.B63u0Dcf.js">
    <link rel="modulepreload" href="/assets/chunks/theme.DB9BSXuu.js">
    <link rel="modulepreload" href="/assets/papers_2019_xue_a.md.C37Fth3J.lean.js">
    <link rel="icon" href="/images/icon.png">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
    <script id="check-mac-os">document.documentElement.classList.toggle("mac",/Mac|iPhone|iPod|iPad/i.test(navigator.platform));</script>
  </head>
  <body>
    <div id="app"><div id="app" class="flex flex-col min-h-screen"><nav class="p-4 mb-4"><div class="flex justify-between items-center w-full px-4 select-none"><a href="/" class="flex items-center no-underline"><img src="/images/icon.png" alt="Logo" class="h-8 w-8 mr-2"><span class="title text-lg font-bold text-custom-orange collapse sm:visible">Bloom Lab</span></a><button class="flex md:hidden text-gray-600"><span class="i-menu h-6 w-6"></span></button><div class="flex bg-custom-soft backdrop-blur-md md:backdrop-blur-none flex-col md:flex-row bg-opacity-75 md:bg-transparent md:flex md:items-center md:px-0 px-3 md:pb-0 pb-10 md:static absolute md:w-auto w-full md:h-auto h-full md:pt-0 pt-10 top-14 z-50 md:flex hidden left-[-100%]"><!--[--><a href="/research/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Research</a><a href="/posts/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Blog</a><a href="/papers/" class="text-custom-orange text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Papers</a><a href="/people/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Team</a><a href="/projects/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Software</a><!--]--></div></div></nav><main class="flex-1 max-w-3xl mx-auto px-4 sm:px-6 lg:max-w-5xl xl:px-0"><div class="prose max-w-none"><div class="text-gray-500 text-lg font-medium flex divide-x divide-gray-300 mb-3"><span class="italic pr-3">Nature genetics</span><span class="pl-3">September 1, 2019</span></div><h1>Reconciling disparate estimates of viral genetic diversity during human influenza infections</h1><div class="text-gray-500 text-lg font-base my-3">Katherine S Xue, Jesse D Bloom</div><span class="text-gray-500 text-base font-medium"> doi:<a href="https://www.nature.com/articles/s41588-019-0349-3" target="_blank">10.1038/s41588-019-0349-3</a></span><!--[--><div style="position:relative;"><div><h2 id="abstract" tabindex="-1">Abstract <a class="header-anchor" href="#abstract" aria-label="Permalink to &quot;Abstract&quot;">​</a></h2><p>To the Editor — A key question in the study of influenza virus evolution is how rapidly viral genetic variation arises within infected humans1,2. Recently, several studies have measured influenza’s within-host genetic diversity in large cohorts of infected humans through high-throughput deep sequencing3,4,5,6 (Supplementary Table 1). These studies disagree in their estimates of influenza’s within-host genetic diversity. In a Nature Genetics letter titled ‘Quantifying influenza virus diversity and transmission in humans’, analyzing a household cohort in Hong Kong, Poon et al.4 have estimated that within-host genetic diversity is high, and 200–250 viral genomes are transmitted between individuals. However, several recent studies conducted in Wisconsin3, Michigan6, and Washington7 that used similar methodologies have estimated lower levels of viral genetic diversity. In particular, the Michigan study has estimated a narrow transmission bottleneck of just one or two viral genomes6. We sought to examine whether technical differences in the underlying deep-sequencing datasets or the methods used to analyze them might explain the disparate estimates of within-host viral genetic diversity. We identified an anomaly in the Hong Kong data that provides a technical explanation for these discrepancies: read pairs from this study are often split between different biological samples, thus indicating that some reads are incorrectly assigned.</p></div></div><!--]--></div></main><footer class="w-full p-4 bg-white text-center text-gray-400 text-sm"><p> Copyright © Jesse Bloom 2024-Present </p></footer></div></div>
    <script>window.__VP_HASH_MAP__=JSON.parse("{\"readme.md\":\"CbtXhfGi\",\"index.md\":\"DIxIrlug\",\"papers_2013_ashenberg.md\":\"k7NZyt5i\",\"papers_2013_gong.md\":\"D9UCtGYJ\",\"papers_2013_hooper.md\":\"fnqWzTL6\",\"papers_2014_bloom_a.md\":\"yEkqLn7r\",\"papers_2014_bloom_b.md\":\"9fq8RI-d\",\"papers_2014_gong.md\":\"yiGjzgzB\",\"people_teagan-mcmahon.md\":\"BXWtxxdB\",\"papers_2020_roop.md\":\"DHOXlbDQ\",\"papers_2014_thyagarajan.md\":\"DmanoLQ7\",\"papers_2015_bloom.md\":\"ZQjWsLYQ\",\"papers_2021_bloom_a.md\":\"BgwgQCFb\",\"papers_2015_doud.md\":\"Db3nSSmr\",\"papers_2021_crawford.md\":\"73lPtRqb\",\"papers_2015_hooper.md\":\"TdDYTkqB\",\"papers_2015_machkovech.md\":\"YZTkTkOE\",\"papers_2016_doud.md\":\"DgEhkLRZ\",\"papers_2016_haddox.md\":\"Be2cMkoz\",\"papers_2016_xue.md\":\"D88zTY0y\",\"papers_2017_ashenberg.md\":\"Gi5tFMPw\",\"papers_2017_hilton.md\":\"CsyPSpdO\",\"papers_2017_xue.md\":\"C7jglluj\",\"papers_2018_dingens.md\":\"CIXRhtJq\",\"papers_2021_eguia.md\":\"Df_OG83V\",\"papers_2021_greaney_a.md\":\"Bw_dzLzS\",\"papers_2021_greaney_b.md\":\"CbB_TBZE\",\"papers_2021_greaney_welsh.md\":\"DqWBA5sC\",\"papers_2021_soh.md\":\"tmTWY-2u\",\"papers_2018_doud.md\":\"Ds38S5fj\",\"papers_2021_starr_greaney_a.md\":\"CgPxihCj\",\"papers_2022_farrell.md\":\"m21iBVU5\",\"papers_2022_greaney_a.md\":\"CUlG-mqq\",\"papers_2022_greaney_b.md\":\"CsGDEhiF\",\"papers_2022_hannon.md\":\"Du6O1-52\",\"papers_2022_starr_a.md\":\"DHoCPElR\",\"papers_2022_starr_greaney_a.md\":\"DUKyOpwH\",\"papers_2022_starr_greaney_b.md\":\"BKFFysFh\",\"papers_2022_yu.md\":\"BRNyGENq\",\"papers_2023_bacsik.md\":\"CWOL-N_J\",\"papers_2023_bloom_a.md\":\"Bggtmrms\",\"papers_2023_bloom_b.md\":\"Cgnr7AS3\",\"papers_2024_hannon.md\":\"BWrzfLSB\",\"papers_2024_larsen.md\":\"Ckey4_2h\",\"papers_2024_loes.md\":\"D55AIq6c\",\"papers_2024_welsh.md\":\"BXZj8zyn\",\"papers_index.md\":\"BwF6OPxe\",\"people_andrea-loes.md\":\"_IrYOmhq\",\"people_andrew-butler.md\":\"C1asfboT\",\"people_arjun-aditham.md\":\"Bl4_g_d3\",\"people_bernadeta-dadonaite.md\":\"DT-9HfR1\",\"people_brendan-larsen.md\":\"BdWHVqXC\",\"people_caelan-radford.md\":\"DxPwImhr\",\"people_caleb-carr.md\":\"Dqiq7oP4\",\"people_caroline-kikawa.md\":\"DeHkM9a8\",\"people_cassandra-simonich.md\":\"X8Eulst4\",\"papers_2018_haddox.md\":\"CWHfGcaJ\",\"people_index.md\":\"DMZS5j4o\",\"people_jenny-ahn.md\":\"CEUjeZc5\",\"papers_2018_hilton.md\":\"CJZJ8zs8\",\"people_jesse-bloom.md\":\"DM0uGz3f\",\"people_lizzie-plender.md\":\"D67eGP9d\",\"people_lucas-kampman.md\":\"u31Wwt91\",\"people_sara-sunshine.md\":\"Bd8X6ffV\",\"people_sheri-harari.md\":\"DMM32Pet\",\"papers_2023_bloom_c.md\":\"CLH9zj-1\",\"papers_2020_loes.md\":\"CHfoqj7I\",\"papers_2018_lee.md\":\"CG8_liny\",\"papers_2018_russell.md\":\"D_5Zy8AY\",\"papers_2020_starr_a.md\":\"DjJ8hGii\",\"papers_2018_xue_a.md\":\"dr-I3_B1\",\"papers_2024_dadonaite_b.md\":\"CkXDrJq7\",\"papers_2018_xue_b.md\":\"rHh1q_L-\",\"papers_2020_xue.md\":\"BYHDpbAn\",\"papers_2019_crawford.md\":\"E6dABOmq\",\"papers_2021_dingens.md\":\"DoTxpM-H\",\"papers_2019_dingens_a.md\":\"B1if6ELj\",\"papers_2019_dingens_b.md\":\"ByB9uv77\",\"papers_2019_machkovech.md\":\"DPncimoB\",\"papers_2019_russell.md\":\"BlSVf-b8\",\"papers_2021_greaney_d.md\":\"BpBl83ef\",\"papers_2021_starr_b.md\":\"CT7lCsvT\",\"papers_2019_xue_a.md\":\"C37Fth3J\",\"papers_2020_crawford.md\":\"L9YNSrjS\",\"papers_2020_dingens.md\":\"BSncmI-R\",\"papers_2020_einav_b.md\":\"mpVO9SeV\",\"papers_2020_gentles.md\":\"DcNqS-YP\",\"papers_2020_hilton.md\":\"DdM4392A\",\"papers_2023_kikawa.md\":\"BI6Xo_EW\",\"people_tim-yu.md\":\"C9UjL41A\",\"people_wenlin-ren.md\":\"TzrRb2TY\",\"people_will-hannon.md\":\"BI6f0jSi\",\"papers_2024_dadonaite_a.md\":\"B7SxfEme\",\"people_xiaohui-ju.md\":\"DTzwYONH\",\"posts_h5-dms.md\":\"GB48UH3v\",\"posts_index.md\":\"DY7viLro\",\"projects_alignparse.md\":\"Bcs1m6OE\",\"projects_dms-vep-pipeline-3.md\":\"B5RIkGxG\",\"projects_dms-viz.md\":\"DNIRscOX\",\"projects_index.md\":\"BCsU3Ywu\",\"projects_neutcurve.md\":\"DDJ_SmEd\",\"projects_polyclonal.md\":\"6Vcxcu4K\",\"projects_seqneut-pipeline.md\":\"CMqldckU\",\"research_index.md\":\"YV_3qyJn\",\"research_research-aims.md\":\"CDvWTbF6\",\"papers_2017_doud.md\":\"DWpy27C2\",\"papers_2017_bloom.md\":\"BROjE1IO\",\"papers_2023_dadonaite_crawford_radford.md\":\"ClFcw1v5\",\"papers_2019_soh.md\":\"CuOq7Qe5\",\"papers_2021_greaney_c.md\":\"fmm-q7rH\",\"papers_2022_gentles.md\":\"DKCVpO5_\",\"papers_2022_greaney_c.md\":\"9q4V1bkM\",\"papers_2024_carr.md\":\"DKhfCz7p\",\"papers_2024_bloom.md\":\"CN1HQTKz\",\"papers_2021_starr_greaney.md\":\"DdyNxImI\",\"papers_2017_dingens.md\":\"BsSLc9VP\",\"papers_2023_radford.md\":\"C_0QkJV-\",\"papers_2019_lee.md\":\"PrjEPIz8\"}");window.__VP_SITE_DATA__=JSON.parse("{\"lang\":\"en-US\",\"dir\":\"ltr\",\"title\":\"Bloom Lab\",\"description\":\"We study the evolution of viruses.\",\"base\":\"/\",\"head\":[],\"router\":{\"prefetchLinks\":true},\"appearance\":false,\"themeConfig\":{},\"locales\":{},\"scrollOffset\":134,\"cleanUrls\":false}");</script>
    
  </body>
</html>