<!DOCTYPE html>
<html lang="en-US" dir="ltr">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <title>alignparse: A Python package for parsing complex features from high-throughput long-read sequencing | Bloom Lab</title>
    <meta name="description" content="We study the evolution of viruses.">
    <meta name="generator" content="VitePress v1.1.0">
    <link rel="preload stylesheet" href="/assets/style.DOJY71uG.css" as="style">
    
    <script type="module" src="/assets/app.DRh-VGvN.js"></script>
    <link rel="modulepreload" href="/assets/chunks/framework.B63u0Dcf.js">
    <link rel="modulepreload" href="/assets/chunks/theme.DF3BNwKE.js">
    <link rel="modulepreload" href="/assets/papers_2019_crawford.md.E6dABOmq.lean.js">
    <meta name="google-site-verification" content="ZIQn6aYwzgvrO-eWI5MYIU5R21R5LnUk95UPij-RXWQ">
    <meta name="keywords" content="Jesse Bloom, Bloom Lab, Fred Hutch Cancer Center, Deep Mutational Scanning, Virus Evolution, Lentiviral Psuedotyping">
    <link rel="icon" href="/images/icon.png">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
    <script id="check-mac-os">document.documentElement.classList.toggle("mac",/Mac|iPhone|iPod|iPad/i.test(navigator.platform));</script>
  </head>
  <body>
    <div id="app"><div id="app" class="flex flex-col min-h-screen"><nav class="p-4 mb-4"><div class="flex justify-between items-center w-full px-4 select-none"><a href="/" class="flex items-center no-underline"><img src="/images/icon.png" alt="Logo" class="h-8 w-8 mr-2"><span class="title text-lg font-bold text-custom-orange collapse sm:visible">Bloom Lab</span></a><button class="flex md:hidden text-gray-600"><span class="i-menu h-6 w-6"></span></button><div class="flex bg-custom-soft backdrop-blur-md md:backdrop-blur-none flex-col md:flex-row bg-opacity-75 md:bg-transparent md:flex md:items-center md:px-0 px-3 md:pb-0 pb-10 md:static absolute md:w-auto w-full md:h-auto h-full md:pt-0 pt-10 top-14 z-50 md:flex hidden left-[-100%]"><!--[--><a href="/research/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Research</a><a href="/posts/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Blog</a><a href="/papers/" class="text-custom-orange text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Papers</a><a href="/people/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Team</a><a href="/projects/" class="text-gray-600 text-xl md:text-lg px-4 py-4 md:py-2 flex items-center justify-center md:flex-none md:items-stretch md:justify-start" data-v-1c4acd03>Software</a><!--]--></div></div></nav><main class="flex-1 max-w-3xl mx-auto px-4 sm:px-6 lg:max-w-5xl xl:px-0"><div class="prose max-w-none"><div class="text-gray-500 text-lg font-medium flex divide-x divide-gray-300 mb-3"><span class="italic pr-3">JOSS</span><span class="pl-3">January 23, 2019</span></div><h1>alignparse: A Python package for parsing complex features from high-throughput long-read sequencing</h1><div class="text-gray-500 text-lg font-base my-3">Katharine HD Crawford, Jesse D Bloom</div><span class="text-gray-500 text-base font-medium"> doi:<a href="https://doi.org/10.21105%2Fjoss.01915" target="_blank">10.21105/joss.01915</a></span><!--[--><div style="position:relative;"><div><h2 id="abstract" tabindex="-1">Abstract <a class="header-anchor" href="#abstract" aria-label="Permalink to &quot;Abstract&quot;">​</a></h2><p>Advances in sequencing technology have made it possible to generate large numbers of long, high-accuracy sequencing reads. For instance, the new PacBio Sequel platform can generate hundreds of thousands of high-quality circular consensus sequences in a single run (Hebert et al., 2018; Rhoads &amp; Au, 2015). Good programs exist for aligning these reads for genome assembly (Chaisson &amp; Tesler, 2012; Li, 2018). However, these long reads can also be used for other purposes, such as sequencing PCR amplicons that contain various features of interest. For instance, PacBio circular consensus sequences have been used to identify the mutations in influenza viruses in single cells (Russell et al, 2019), or to link barcodes to gene mutants in deep mutational scanning (Matreyek et al., 2018). For such applications, the alignment of the sequences to the targets may be fairly trivial, but it is not trivial to then parse specific features of interest (such as mutations, unique molecular identifiers, cell barcodes, and flanking sequences) from these alignments.</p></div></div><!--]--></div></main><footer class="w-full p-4 bg-white text-center text-gray-400 text-sm"><p> Copyright © Jesse Bloom 2024-Present </p></footer></div></div>
    <script>window.__VP_HASH_MAP__=JSON.parse("{\"papers_2015_machkovech.md\":\"YZTkTkOE\",\"papers_2014_gong.md\":\"yiGjzgzB\",\"papers_2014_thyagarajan.md\":\"DmanoLQ7\",\"papers_2014_bloom_b.md\":\"9fq8RI-d\",\"papers_2015_bloom.md\":\"ZQjWsLYQ\",\"papers_2015_doud.md\":\"Db3nSSmr\",\"papers_2015_hooper.md\":\"TdDYTkqB\",\"papers_2021_greaney_d.md\":\"BpBl83ef\",\"papers_2021_greaney_welsh.md\":\"DqWBA5sC\",\"papers_2013_gong.md\":\"D9UCtGYJ\",\"papers_2018_xue_b.md\":\"rHh1q_L-\",\"papers_2021_eguia.md\":\"Df_OG83V\",\"papers_2018_xue_a.md\":\"dr-I3_B1\",\"papers_2021_crawford.md\":\"73lPtRqb\",\"papers_2016_doud.md\":\"DgEhkLRZ\",\"papers_2016_xue.md\":\"D88zTY0y\",\"papers_2021_greaney_a.md\":\"Bw_dzLzS\",\"papers_2019_dingens_a.md\":\"B1if6ELj\",\"papers_2021_greaney_b.md\":\"CbB_TBZE\",\"papers_2017_ashenberg.md\":\"Gi5tFMPw\",\"papers_2017_dingens.md\":\"BsSLc9VP\",\"papers_2017_doud.md\":\"DWpy27C2\",\"papers_2019_dingens_b.md\":\"ByB9uv77\",\"papers_2017_bloom.md\":\"BROjE1IO\",\"papers_2019_russell.md\":\"BlSVf-b8\",\"papers_2018_dingens.md\":\"CIXRhtJq\",\"papers_2020_crawford.md\":\"L9YNSrjS\",\"papers_2019_xue_a.md\":\"C37Fth3J\",\"papers_2020_dingens.md\":\"BSncmI-R\",\"papers_2018_doud.md\":\"Ds38S5fj\",\"papers_2018_hilton.md\":\"CJZJ8zs8\",\"papers_2018_lee.md\":\"CG8_liny\",\"papers_2020_hilton.md\":\"DdM4392A\",\"papers_2020_roop.md\":\"DHOXlbDQ\",\"papers_2018_russell.md\":\"D_5Zy8AY\",\"papers_2020_gentles.md\":\"DcNqS-YP\",\"papers_2018_haddox.md\":\"CWHfGcaJ\",\"papers_2019_lee.md\":\"PrjEPIz8\",\"index.md\":\"DIxIrlug\",\"papers_2016_haddox.md\":\"Be2cMkoz\",\"papers_2013_ashenberg.md\":\"k7NZyt5i\",\"papers_2021_dingens.md\":\"DoTxpM-H\",\"papers_2021_soh.md\":\"tmTWY-2u\",\"papers_2020_starr_a.md\":\"DjJ8hGii\",\"papers_2020_xue.md\":\"BYHDpbAn\",\"papers_2021_bloom_a.md\":\"BgwgQCFb\",\"people_brendan-larsen.md\":\"BdWHVqXC\",\"people_caelan-radford.md\":\"DxPwImhr\",\"people_caleb-carr.md\":\"Dqiq7oP4\",\"people_wenlin-ren.md\":\"DLUiLLWb\",\"people_lizzie-plender.md\":\"D67eGP9d\",\"people_lucas-kampman.md\":\"B3dR_q9D\",\"people_sara-sunshine.md\":\"Cqk5Xzyb\",\"people_caroline-kikawa.md\":\"ChtxsWhY\",\"people_sheri-harari.md\":\"DMM32Pet\",\"papers_2023_kikawa.md\":\"BI6Xo_EW\",\"papers_2023_radford.md\":\"C_0QkJV-\",\"people_cassandra-simonich.md\":\"NkwF4VVo\",\"people_tim-yu.md\":\"C9UjL41A\",\"people_teagan-mcmahon.md\":\"CT00y3Cm\",\"papers_2019_machkovech.md\":\"DPncimoB\",\"papers_2024_bloom.md\":\"CN1HQTKz\",\"projects_seqneut-pipeline.md\":\"CMqldckU\",\"projects_polyclonal.md\":\"6Vcxcu4K\",\"people_xiaohui-ju.md\":\"DTzwYONH\",\"projects_index.md\":\"BCsU3Ywu\",\"papers_2021_starr_greaney_a.md\":\"CgPxihCj\",\"projects_alignparse.md\":\"Bcs1m6OE\",\"papers_2021_starr_greaney.md\":\"DdyNxImI\",\"papers_2024_dadonaite_a.md\":\"B7SxfEme\",\"papers_2024_dadonaite_b.md\":\"CkXDrJq7\",\"projects_neutcurve.md\":\"DDJ_SmEd\",\"papers_2019_soh.md\":\"CuOq7Qe5\",\"people_andrew-butler.md\":\"C1asfboT\",\"papers_2021_greaney_c.md\":\"fmm-q7rH\",\"papers_2024_loes.md\":\"D55AIq6c\",\"readme.md\":\"DpgXHpl_\",\"papers_2019_crawford.md\":\"E6dABOmq\",\"papers_2020_loes.md\":\"CHfoqj7I\",\"papers_2017_hilton.md\":\"CsyPSpdO\",\"papers_2014_bloom_a.md\":\"yEkqLn7r\",\"papers_2020_einav_b.md\":\"mpVO9SeV\",\"research_index.md\":\"YV_3qyJn\",\"papers_2013_hooper.md\":\"fnqWzTL6\",\"papers_2017_xue.md\":\"C7jglluj\",\"research_research-aims.md\":\"CDvWTbF6\",\"people_index.md\":\"DMZS5j4o\",\"people_jenny-ahn.md\":\"CEUjeZc5\",\"posts_index.md\":\"DY7viLro\",\"papers_2022_starr_greaney_a.md\":\"DUKyOpwH\",\"papers_2022_greaney_c.md\":\"9q4V1bkM\",\"projects_dms-vep-pipeline-3.md\":\"B5RIkGxG\",\"papers_2022_hannon.md\":\"Du6O1-52\",\"papers_2022_starr_greaney_b.md\":\"BKFFysFh\",\"papers_2022_yu.md\":\"BRNyGENq\",\"papers_2024_hannon.md\":\"BWrzfLSB\",\"papers_2023_bacsik.md\":\"CWOL-N_J\",\"papers_2024_larsen.md\":\"Ckey4_2h\",\"papers_2022_gentles.md\":\"DKCVpO5_\",\"papers_2021_starr_b.md\":\"CT7lCsvT\",\"papers_2022_greaney_a.md\":\"CUlG-mqq\",\"papers_2022_greaney_b.md\":\"CsGDEhiF\",\"people_jesse-bloom.md\":\"DM0uGz3f\",\"people_bernadeta-dadonaite.md\":\"B_bttk2E\",\"papers_2024_welsh.md\":\"BXZj8zyn\",\"people_arjun-aditham.md\":\"Bl4_g_d3\",\"papers_2022_farrell.md\":\"m21iBVU5\",\"papers_2023_bloom_b.md\":\"Cgnr7AS3\",\"papers_index.md\":\"BwF6OPxe\",\"papers_2023_bloom_c.md\":\"CLH9zj-1\",\"people_andrea-loes.md\":\"_IrYOmhq\",\"papers_2023_dadonaite_crawford_radford.md\":\"ClFcw1v5\",\"papers_2024_carr.md\":\"DKhfCz7p\",\"projects_dms-viz.md\":\"DNIRscOX\",\"people_will-hannon.md\":\"BI6f0jSi\",\"papers_2023_bloom_a.md\":\"Bggtmrms\",\"posts_h5-dms.md\":\"GB48UH3v\",\"papers_2022_starr_a.md\":\"DHoCPElR\"}");window.__VP_SITE_DATA__=JSON.parse("{\"lang\":\"en-US\",\"dir\":\"ltr\",\"title\":\"Bloom Lab\",\"description\":\"We study the evolution of viruses.\",\"base\":\"/\",\"head\":[],\"router\":{\"prefetchLinks\":true},\"appearance\":false,\"themeConfig\":{},\"locales\":{},\"scrollOffset\":134,\"cleanUrls\":false}");</script>
    
  </body>
</html>