---
layout: paper
title: "An experimentally determined evolutionary model dramatically improves phylogenetic fit"
date: "2014-08-01"
authors: 
    - "Jesse D Bloom"
journal: "Molecular Biology and Evolution"
doi: "10.1093/molbev/msu173"
link: "https://academic.oup.com/mbe/article/31/8/1956/2925795"
image: "/assets/papers/bloom_2014b.jpg"
keywords:
    - "Phylogenetics"
---

## Abstract

All modern approaches to molecular phylogenetics require a quantitative model for how genes evolve. Unfortunately, existing evolutionary models do not realistically represent the site-heterogeneous selection that governs actual sequence change. Attempts to remedy this problem have involved augmenting these models with a burgeoning number of free parameters. Here, I demonstrate an alternative: Experimental determination of a parameter-free evolutionary model via mutagenesis, functional selection, and deep sequencing. Using this strategy, I create an evolutionary model for influenza nucleoprotein that describes the gene phylogeny far better than existing models with dozens or even hundreds of free parameters. Emerging high-throughput experimental strategies such as the one employed here provide fundamentally new information that has the potential to transform the sensitivity of phylogenetic and genetic analyses.
