---
layout: paper
title: "Improving pandemic influenza risk assessment"
date: "2014-10-16"
authors: 
    - "Colin A Russell"
    - "Peter M Kasson"
    - "Ruben O Donis"
    - "Steven Riley"
    - "John Dunbar"
    - "Andrew Rambaut"
    - "Jason Asher"
    - "Stephen Burke"
    - "C Todd Davis"
    - "Rebecca J Garten"
    - "Sandrasegaram Gnanakaran"
    - "Simon I Hay"
    - "Sander Herfst"
    - "Nicola S Lewis"
    - "James O Lloyd-Smith"
    - "Catherine A Macken"
    - "Sebastian Maurer-Stroh"
    - "Elizabeth Neuhaus"
    - "Colin R Parrish"
    - "Kim M Pepin"
    - "Samuel S Shepard"
    - "David L Smith"
    - "David L Suarez"
    - "Susan C Trock"
    - "Marc-Alain Widdowson"
    - "Dylan B George"
    - "Marc Lipsitch"
    - "Jesse D Bloom"
journal: "Elife"
doi: "10.7554/eLife.03883.001"
link: "https://elifesciences.org/articles/3883"
image: "/assets/papers/starr_greany_2020.png"
keywords:
    - "Influenza"
    - "Policy"
---

## Abstract

Assessing the pandemic risk posed by specific non-human influenza A viruses is an important goal in public health research. As influenza virus genome sequencing becomes cheaper, faster, and more readily available, the ability to predict pandemic potential from sequence data could transform pandemic influenza risk assessment capabilities. However, the complexities of the relationships between virus genotype and phenotype make such predictions extremely difficult. The integration of experimental work, computational tool development, and analysis of evolutionary pathways, together with refinements to influenza surveillance, has the potential to transform our ability to assess the risks posed to humans by non-human influenza viruses and lead to improved pandemic preparedness and response.
